doc/specs/userSettings.md

```diff
@@ -0,0 +1,82 @@
+# User Settings
+
+## Metadata
+
+Author: Tick
+
+### Changelog
+
+**2015-07-28**
+
+* Initial version
+
+## Table of contents
+
+1. Motivation
+1. Business rules
+1. Use cases
+
+## Motivation
+
+Provides a means for users of the site to configure and personalise their usage of the site.
+
+## Business rules.
+
+ - A user can only edit the own settings
+ - Administrators with the right permissions can edit another user's settings.
```

Comment:
What about users with the settings admin permission trying to edit the settings of a user above them? i.e. a mod trying to edit an admin's settings.